code/src/index.css

```diff
@@ -1,13 +1,169 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  margin: 20px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+header {
+  margin: 0 auto;
+}
+
+.header {
+  margin-bottom: 0px;
+  align-self: center;
+  border-bottom: 1px solid black;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 350px;
+  min-height: 550px;
+  border-radius: 2%;
+  border: 1px solid black;
+  background-color: rgba(242, 181, 90, 0.4);
+
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.addtodo-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  width: 350px;
+  padding: 1em;
+  height: 80px;
+  position: absolute;
+  bottom: 0;
+  z-index: 1;
+}
+
+input {
+  margin: 0.5em;
+  padding: 0.5em;
+  border: rgb(34, 27, 27);
+}
+
+.checkbox {
+  background-color: none;
+  cursor: default;
+  appearance: auto;
+  box-sizing: border-box;
+  padding: none;
+  border: none;
+  align-self: center;
+}
+
+.delete-btn {
+  max-height: 20px;
+  align-self: center;
+}
+
+.add-btn {
+  align-self: center;
+  appearance: auto;
+  /* font-style: ;
+  font-variant-ligatures: ;
+  font-variant-caps: ;
+  font-variant-numeric: ;
+  font-variant-east-asian: ;
+  font-weight: ;
+  font-stretch: ; */
+  font-size: 1rem;
+  /* font-family: ; */
+  text-rendering: auto;
+  color: -internal-light-dark(black, white);
+  letter-spacing: normal;
+  word-spacing: normal;
+  line-height: normal;
+  text-transform: none;
+  text-indent: 0px;
+  text-shadow: none;
+  display: inline-block;
+  text-align: center;
+  align-items: flex-start;
+  cursor: default;
+  box-sizing: border-box;
+  background-color: transparent;
+  margin: 0em;
+  padding: 10px 15px;
+  border-width: 2px;
+  border-style: ridge;
+  border-color: black;
+  border-image: initial;
```

Comment:
Looking at your button in devtools there's some styling that doesn't change the appereance of your button. Maybe this could , be deleted. As well as the code you comment out already? :) 
![todo-jakob2](https://user-images.githubusercontent.com/81567468/143626454-2eb98d88-5a6b-426f-a63c-df6098a3dc19.PNG)